integration_tests/serverless.yml

```diff
@@ -86,6 +96,15 @@ functions:
       DD_ENHANCED_METRICS: false
       DD_LAMBDA_HANDLER: src/metricPython.metric
 
+  no-enhanced-metric-go:
```

Comment:
This function is called "no-enhanced-metric-go" but the handler is testing without the Lambda Layer. I think what we are really testing here is without the Lambda layer, so let's name the function based on that.

Reply:
👍